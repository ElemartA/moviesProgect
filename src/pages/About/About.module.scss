.container {
  display: block;
  justify-content: center;
  font-size: 30px;
  max-width: 1200px;
  margin: 0 auto;
  font-family: "Cormorant Garamond", serif;
  flex-wrap: wrap;
  .container__title{
    font-weight: bold;
  }
}
