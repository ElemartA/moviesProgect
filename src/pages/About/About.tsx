import React from "react";
import s from "./About.module.scss";

const About = () => (
  <div data-testid="aboutPage" className={s.container}>
    <h2 className={s.container__title}>Kinopoisk Api Unofficial</h2>
    <p>Неофициальное api для доступа к актуальной информации кинопоиска.</p>
    <ol>
      <li>
        Апи для тех кто хочет получать все данные о фильмах максимально быстро в
        удобном формате.
      </li>
      <li>
        На кинопоиск каждый день добавляются сотни данных о новинках
        кинематографа, актерах, режиссерах.
      </li>
      <li>
        Пока Вы отдыхаете или занимаетесь своими проектами - мы следим за
        обновлениями на кинопоиск и загружаем данные в api
      </li>
    </ol>
  </div>
);

export default About;
