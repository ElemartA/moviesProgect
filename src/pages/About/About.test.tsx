import React from "react";
import { render, screen } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import About from "./About";

describe("About", () => {
  it("renders About component", () => {
    const { asFragment } = render(
      <BrowserRouter>
        <About />
      </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
    screen.debug();
    expect(screen.getByRole("heading")).toBeInTheDocument();
    expect(screen.getByRole("heading")).toHaveTextContent(
      "Kinopoisk Api Unofficial"
    );
    expect(screen.queryByText(/Апи для тех кто хочет/i)).toBeInTheDocument();
    expect(screen.queryByText(/Пока Вы отдыхаете/i)).toBeInTheDocument();
    expect(screen.queryByText(/О нас/i)).toBeNull();
  });
});
