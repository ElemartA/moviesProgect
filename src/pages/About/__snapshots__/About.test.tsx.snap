// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders About component 1`] = `
<DocumentFragment>
  <div
    class="container"
    data-testid="aboutPage"
  >
    <h2
      class="container__title"
    >
      Kinopoisk Api Unofficial
    </h2>
    <p>
      Неофициальное api для доступа к актуальной информации кинопоиска.
    </p>
    <ol>
      <li>
        Апи для тех кто хочет получать все данные о фильмах максимально быстро в удобном формате.
      </li>
      <li>
        На кинопоиск каждый день добавляются сотни данных о новинках кинематографа, актерах, режиссерах.
      </li>
      <li>
        Пока Вы отдыхаете или занимаетесь своими проектами - мы следим за обновлениями на кинопоиск и загружаем данные в api
      </li>
    </ol>
  </div>
</DocumentFragment>
`;
